frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -284,26 +284,22 @@ const ConsolePlugins: React.FC<ConsolePluginsProps> = ({ csvPlugins, trusted })
     verb: 'patch',
     name: CONSOLE_OPERATOR_CONFIG_NAME,
   });
-  const csvPluginsCount = csvPlugins.length;
 
   return (
     <>
       {consoleOperatorConfig && canPatchConsoleOperatorConfig && (
         <dl className="co-clusterserviceversion-details__field">
-          <dt>{t('olm~Console plugin', { count: csvPluginsCount })}</dt>
+          <dt>{t('olm~Console plugin')}</dt>
```

Comment:
We still want the pluralization if there are more than one?

Reply:
yes, you are right
updated~